sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,10 @@
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (16336, 51971, 52010);
+INSERT INTO `spell_script_names` (`spell_id`,`ScriptName`) VALUES
+(16336, 'spell_stratholme_haunting_phantoms'),
+(51971, 'spell_scourge_disguise_instability'),
+(52010, 'spell_scourge_disguise_expiring');
+
+DELETE FROM `spell_linked_spell` WHERE `spell_trigger` = 51966;
+INSERT INTO `spell_linked_spell` (`spell_trigger`, `spell_effect`, `type`, `comment`) VALUES
+(51966, 51971, 1, 'Scourge Disguise applies Scourge Disguise Instability');
```

Comment:
I think this is considered hacky and will not be accepted 😄

Reply:
this table should be dropped... :)

Reply:
In cases that is: Spell 1 -> Trigger Spell 2, it dont even check if Spell1 realy hit target ._.

Reply:
Yeah this table blows

Reply:
I'm not gonna complain that it hasn't been nuked already in that case, but could you at least make a note of it on https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130132/spell+linked+spell?